### PR TITLE
Disable Firebase Performance for development builds

### PR DESCRIPTION
**Pull request by vinaysshenoy**

### CODE REVIEW

**Review by Honey14 — APPROVED** [review]
Looks good

File: app/src/debug/AndroidManifest.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     tools:replace="android:name,android:icon">
 
     <meta-data
-      android:name="firebase_performance_logcat_enabled"
+      android:name="firebase_performance_collection_deactivated"
```
[the remark this answers is not in the document]
Comment (vinaysshenoy) [reply app/src/debug/AndroidManifest.xml:13]: Yup. that is correct. This just means that Firebase Performance Monitoring will not be active in dev builds.

**Pull request merged**